### PR TITLE
E-1070 feat: timezone metric

### PR DESCRIPTION
**Pull request by ifiri**

### CODE REVIEW

**Comment by lamartire** [comment]
As I said in #323 - install and reuse utilities from new version of `@endpass/utils`.
Also, you can collect other metrics and send it. What do you think, @kazurus?

**Comment by kazurus** [comment]
yes, we can

**Comment by kazurus** [comment]
@ifiri please, add ip as well

**Comment by ifiri** [comment]
@kazurus do you about rebase? this is possible

if you about manual edit, I guess, it's not a good practice, all changes between branches should be done only via merge \ rebase

**Comment by kazurus** [comment]
@ifiri in this pr we have timezone, fingerprint. Need webrtc ip from our utils. We don't have a branch for webrtc

**Comment by ifiri** [comment]
@kazurus as I know, there was PR with IP collecting, right? are about it? is it merged?

**Comment by ifiri** [comment]
@kazurus figured out, it will added as in vault

**Comment by ifiri** [comment]
@kazurus done

**Comment by kazurus** [comment]
@lamartire

**Comment by kazurus** [comment]
@ifiri :recycle:

File: src/constants/index.js [hunk 1]
```diff
@@ -169,6 +169,19 @@ export const DOC_TYPES = {
...
 };
 
+/**
+ * @type {{
+    USER_BROWSER_FINGERPRINT: 'user:browser:fingerprint',
+    USER_IP_WEBRTC: 'user:ip:webrtc',
+    USER_BROWSER_TIMEZONE: 'user:browser:timezone',
+  }}
+ */
+export const METRIC_TYPES = {
```
Comment (lamartire) [inline src/constants/index.js:179]: Place this constant near the scoring store, like in the vault

File: src/store/modules/RiskScoringModule.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+import riskScoringService from '@/service/riskScoring';
+
+@Module({ generateMutationSetters: true })
+class RiskScoringModule extends VuexModule {
+  /**
+   * @return {Promise<void>}
+   */
+  @Action
+  async sendUserMetrics() {
+    await riskScoringService.sendUserMetrics();
```
Comment (lamartire) [inline src/store/modules/RiskScoringModule.js:11]: Do we really need module instead controller? Module needs when we have state, but this module has not

File: tests/unit/mocks/service/riskScoring.mock.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import riskScoringService from '@/service/riskScoring';
+
+jest.mock('@/service/riskScoring', () => {
+  return {
+    sendUserMetrics: jest.fn(),
```
Comment (lamartire) [inline tests/unit/mocks/service/riskScoring.mock.js:5]: Add `.mockResolvedValue()`

File: src/store/modules/RiskScoringModule.js (hunk 2 above)
Comment (ifiri) [reply src/store/modules/RiskScoringModule.js:11]: Commonly we use only private controllers, if something global - then modules. This is global. One module instead 3-4 private controllers with same code.

Comment (lamartire) [reply src/store/modules/RiskScoringModule.js:11]: We can also use global controller. @kazurus am I right?

Comment (ifiri) [reply src/store/modules/RiskScoringModule.js:11]: If it is acceptable, then yes 🙂 but generally we have only WalletController in auth, all remains - private or store. So this is uncommon approach.

Comment (kazurus) [reply src/store/modules/RiskScoringModule.js:11]: We should use global controller instead store. We have similar in Vault.

Comment (dudiq) [reply src/store/modules/RiskScoringModule.js:11]: what about don't use store or global controller.
and use just single `scoringService` and call them not from components, but from controllers (backController, frontOnlyController) in places, where we need `sendUserMetrics`?
@endpass/frontend ?

Comment (kazurus) [reply src/store/modules/RiskScoringModule.js:11]: Good idea

Comment (ifiri) [reply src/store/modules/RiskScoringModule.js:11]: Agreed

**Comment by ifiri** [comment]
@lamartire all fixed and tested

**Comment by kazurus** [comment]
@ifiri :recycle:

**Comment by ifiri** [comment]
@kazurus done

**Pull request merged**